### PR TITLE
Scalyr support

### LINKED ISSUE #17: Add support for scalyr
Add support for scalyr agent.

### PR DESCRIPTION
**Pull request by mohabusama**
- Now supports scalyr and appdynamics (can be set as command line arg or env variable)

Resolve #17

### CODE REVIEW

File: k8s_log_watcher/agents/scalyr.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            raise RuntimeError('Scalyr watcher agent initialization failed. Env variables WATCHER_SCALYR_API_KEY and '
+                               'WATCHER_SCALYR_DEST_PATH must be set.')
+
+        self.config_path = os.environ.get('WATCHER_SCALYR_CONFIG_PATH', SCALYR_CONFIG_PATH)
+        if not os.path.exists(os.path.dirname(self.config_path)):
+            raise RuntimeError(
+                'Scalyr watcher agent initialization failed. {} config path does not exist.'.format(
+                    self.config_path))
+
+        self.monitor_syslog = os.environ.get('WATCHER_SCALYR_SYSLOG_TCP_PORT', 0)
```
Comment (hjacobs) [inline k8s_log_watcher/agents/scalyr.py]: we don't need the syslog stuff anymore, right?

Comment (mohabusama) [inline k8s_log_watcher/agents/scalyr.py]: True! I actually need to add ``journald`` monitor instead.

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**